### PR TITLE
Setup @twilio/webrtc to use release-tool

### PR DESCRIPTION
**Pull request by markandrus**
@manjeshbhargav @ryan-rowland

### CODE REVIEW

**Comment by ryan-rowland** [comment]
👍 LGTM

File: .release.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        "git remote set-url origin \"https://${GH_TOKEN}@${GH_REF}\"",
+        "git rebase HEAD ${BRANCH}",
+        "git push origin ${BRANCH} --tags"
+      ]
+    },
+    "development": {
+      "commands": [
+        "node ./node_modules/.bin/release --bump ${RELEASE_VERSION} ${DEVELOPMENT_VERSION}",
+        "npm run clean",
+        "git add package.json bower.json",
```
Comment (manjeshbhargav) [inline .release.json]: @markandrus No need of `bower` here?

**Comment by manjeshbhargav** [comment]
@markandrus One comment, otherwise looks good.

**Comment by manjeshbhargav** [comment]
+1

**Pull request merged**